test/spec/util/convertBasicSearch.spec.js

```diff
@@ -76,20 +106,36 @@
                     });
                 });
                 describe('When invoked with a multi-word string with extraneous whitespace', function () {
-                    expect(convertBasicSearch('field', '  forty    two ')).to.deep.equal({
-                        operator: 'AND',
-                        children: [
-                            {
-                                field: 'field',
-                                comparator: 'contains',
-                                value: 'forty'
-                            },
-                            {
-                                field: 'field',
-                                comparator: 'contains',
-                                value: 'two'
-                            }
-                        ]
+                    it('Returns the correct query', function () {
```

Comment:
This was causing the issues with the tests falling over completely.  It was only an issue when this particular test failed.  Assertions always need to be in an `it` function, never directly in a `describe` function.